rest_api/src/main/java/tqs/cloudit/domain/persistance/User.java

```diff
@@ -43,7 +44,7 @@
     @JsonIgnore
     private Set<JobOffer> acceptedOffers = new HashSet();
 
-    @ManyToMany
+    @ManyToMany(cascade = {CascadeType.ALL})
```

Comment:
I think this may cause some problems. In the case 2 users have the same interest areas and you delete 1 of them, are the areas removed from the database? Does the other user lose all it's areas?

Reply:
See my next commit please